pallet-fuso-foundation/src/lib.rs

```diff
@@ -26,29 +36,32 @@ pub mod pallet {
     use sp_runtime::traits::{Saturating, Zero};
 
     pub type BalanceOf<T> = <T as pallet_balances::Config>::Balance;
-    pub type IdentifierOf<T> = <T as pallet_balances::Config>::ReserveIdentifier;
 
     #[pallet::config]
     pub trait Config: frame_system::Config + pallet_balances::Config {
```

Comment:
we'd prefer using associate type rather than inheriting traits, 
```
pub type BalanceOf<T> = <<T as Config>::Currency as Currency<<T as frame_system::Config>::AccountId>>::Balance;
```

type Currency: Currency;